state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -1184,7 +1180,7 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 			fees.saturating_accrue(fee_estimate);
 		});
 
-		remaining_amount
```

Comment:
I'm not sure this is what we want. The fee estimate is in terms of the GAS_ASSET, ie. if someone sends USDC, it will be in terms of ETH. 

I think we want to return (remaining_amount, available_amount - remaining_amount).

Or maybe we want both 🤔 

(https://linear.app/chainflip/issue/PRO-1118/expose-fees-in-ingressegress-asset-amount)

Reply:
Updated to return `(remaining_amount, available_amount - remaining_amount)` for now.